<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yearRange</class>
 <widget class="QDialog" name="yearRange">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Year</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>End Year</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>9999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEnd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>9999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>124</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filterScope">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Graphs Only</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Graphs Only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Records and Graphs</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelStart_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
